### ISSUE TITLE
decode时部分数据会丢失

### ISSUE THREAD

**Issue by NCZkevin** [jiesutd/NCRFpp#77]
我在使用框架完成中文NER任务时，数据是用BIO格式标注的，训练一切正常，但是decode的时候生成的文件有部分数据会丢失。比如一个5219行的文件，decode之后只剩下前3429行。一个50W行的文件decode之后的文件只有30多W行，decode过程没有任何报错信息，请问有可能是什么原因造成的？是数据的问题吗？感谢您

下面是train的config文件
### I/O ###
train_dir=data/train.data
dev_dir=data/dev.data
test_dir=data/test.data
model_dir=data/lstmcrf
word_emb_dir=data/sample.word.emb

#raw_dir=
#decode_dir=
#dset_dir=
#load_model_dir=
#char_emb_dir=

norm_word_emb=False
norm_char_emb=False
number_normalized=True
seg=True
word_emb_dim=50
char_emb_dim=30

###NetworkConfiguration###
use_crf=True
use_char=False
word_seq_feature=LSTM
char_seq_feature=CNN
#feature=[POS] emb_size=20
#feature=[Cap] emb_size=20
#nbest=1

###TrainingSetting###
status=train
optimizer=SGD
iteration=50
batch_size=256
ave_batch_loss=False

###Hyperparameters###
cnn_layer=4
char_hidden_dim=50
hidden_dim=200
dropout=0.5
lstm_layer=1
bilstm=True
learning_rate=0.015
lr_decay=0.05
momentum=0
l2=1e-8
gpu=True
#clip=
下面是decode的config文件
### Decode ###
status=decode
raw_dir=data/33.data
nbest=0
decode_dir=data/33.out
dset_dir=data/lstmcrf.dset
load_model_dir=data/lstmcrf.20.model

**Comment by jiesutd** [comment]
It may be caused by the long sentences in your datasets. In default, the sentence length longer than 250 words would be filtered out. You can set the like `MAX_SENTENCE_LENGTH=300` to enable the model to take longer sentences. Or you can manually change the default max sentence length here: https://github.com/jiesutd/NCRFpp/blob/b6c7df699b667d8ed29cea2d710995d9f8cbcad2/utils/data.py#L24

Otherwise, you can break the long sentence into short pieces.
